### DISCUSSION TITLE
How to add more then one name?

### DISCUSSION THREAD

**Discussion by abdulmateentechbits** [react-hook-form/react-hook-form#9403] [Q&A] [answered]
How can I add more then one name like below, 
this cause an error,
const { fields, append, remove } = useFieldArray({ control, name: ['dr_heads' ,'cr_heads' ]});
How can i deal with two separate dynamic form fields

**Comment by bluebill1049** [comment, marked as answer]
just have two sperate `useFieldArray` hooks.

**Reply by abdulmateentechbits** [reply]
Like this? 
const {
    fields,
    append,
    remove
  } = useFieldArray({ control, name: "food" });
  const {
    fields,
    append,
    remove
  } = useFieldArray({ control, name: "drinks" });

**Comment by abdulmateentechbits** [comment]
Thanks this is worked for me
 const {
    fields: foodFields,
    append: foodAppend,
    remove: foodRemove
  } = useFieldArray({ control, name: "food" });
  const {
    fields: drinkFields,
    append: drinkAppend,
    remove: drinkRemove
  } = useFieldArray({ control, name: "drinks" });
